### DISCUSSION TITLE
Thank you for the update!

### DISCUSSION THREAD

**Discussion by jffaust** [chrisgoringe/cg-use-everywhere#197] [General discussion]
Hey @chrisgoringe , I couldn't find you on social media like X/Twitter so I'm posting here.

I just played with the nodes in ComfyUI and noticed there was now autocompletion or suggestions for the Anything Everywhere node and it's a really nice addition.

Thanks for the great work :)

**Comment by chrisgoringe** [comment]
Thanks! I'm not on X, but you can get me on discord, fwiw, as  chris.goringe
